src/python/grpcio/build.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2016, Google Inc.
```

Comment:
I think you might be able to simplify this:

If my understanding is correct, this is a workaround the windows command line character limit. What if instead of monkeypatching the entire link operation, you just monkeypatch the spawn() operation?